admin.code-workspace

```diff
@@ -2,39 +2,46 @@
 	"folders": [
     {
       "name": "admin",
-      "path": "apps\\admin"
+      "path": "apps/admin"
+    },
+    {
+      "name": "tx-etransfer",
+      "path": "libs/tx-etransfer"
     },
     {
       "name": "shared",
-      "path": "libs\\shared"
+      "path": "libs/shared"
     },
     {
       "name": "tx-reconciliation",
-      "path": "libs\\tx-reconciliation"
+      "path": "libs/tx-reconciliation"
     },
     {
       "name": "tx-gmail",
-      "path": "libs\\tx-gmail"
+      "path": "libs/tx-gmail"
     },
     {
-      "name": "utils-ts",
-      "path": "libs\\utils-ts"
+      "name": "broker-db",
+      "path": "libs/broker-db"
     },
     {
-      "name": "rbcapi",
-      "path": "libs\\rbcapi"
+      "name": "utils-ts",
+      "path": "libs/utils-ts"
     },
     {
       "name": "store",
-      "path": "libs\\store"
+      "path": "libs/store"
     },
     {
       "name": "logging",
-      "path": "libs\\logging"
+      "path": "libs/logging"
     },
     {
       "name": "contract",
-      "path": "libs\\contract"
+      "path": "libs/contract"
+    },
+    {
+      "path": "libs/signers"
```

Comment:
No name for this one?

Reply:
huh, weird.  It shows up with a name.  I've never explicitly named the other ones, so maybe this is a new feature in VS Code?